src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+@import "../styles/utils/var.scss";
+@import "../styles/mixin/mixin.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  width: 100%;
+  padding-left: 25%;
+}
+
+.calendar {
+  display: flex;
+  align-content: center;
+  gap: $cellGap;
+  padding: $padd;
+  max-width: $weekWidth;
+  margin: 10% auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.